Makefile

```diff
@@ -69,9 +69,11 @@ export SUBPACKAGES := $(SUBPACKAGES)
 
 KIND_VERSION = v0.15.0
 # dependency for up
-UP_VERSION = v0.20.0
+UP_VERSION = v0.28.0
 UP_CHANNEL = stable
-UPTEST_VERSION = v0.8.0
+UPTEST_VERSION = v0.11.1
+UPTEST_LOCAL_VERSION = v0.12.0-9.gac371c9
+UPTEST_LOCAL_CHANNEL = main
```

Comment:
nit: We may consider using after releaseing the uptest with the import fix. I am not proposing to do this in this PR. I want to only note this.

Reply:
Yes, agreed. Let's revisit this once https://github.com/upbound/build/pull/255 is merged. Thanks for the note.